app/src/main/java/com/ecosystem/kin/app/NativeOfferActivity.java

```diff
@@ -0,0 +1,15 @@
+package com.ecosystem.kin.app;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+
+public class NativeOfferActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.native_offer_activity);
```

Comment:
would be nice to show toast or UI of the offer that was just pressed - even better will be able to completed a native spend , wait for callback and then remove offer from market place if completed successful - can open a task for it